.infrastructure/60_cdn.tf

```diff
@@ -110,3 +110,52 @@ resource "aws_cloudfront_distribution" "website" {
     ssl_support_method             = var.use_custom_certificate ? "sni-only" : null
   }
 }
+
+## CDN to Medialibrary for CMS Strapi
+module "cloudfront_cms" {
+  source = "git::https://github.com/terraform-aws-modules/terraform-aws-cloudfront.git?ref=ed0f1f983f606304e00ad9f48399bd2fe0b79233" # v3.2.2
+
+  create_origin_access_identity = true
+  origin_access_identities = {
+    s3_cms = "Identity to access S3 bucket."
+  }
+
+  origin = {
+    s3_one = {
+      domain_name = module.s3_bucket_cms.s3_bucket_bucket_regional_domain_name
+      s3_origin_config = {
+        origin_access_identity = "s3_cms"
+      }
+    }
+  }
+
+  enabled         = true
+  is_ipv6_enabled = true
+  comment         = "CloudFront distribution for the medialibrary cms."
+
+  default_cache_behavior = {
+    allowed_methods        = ["GET", "HEAD", "OPTIONS", "PUT", "POST", "PATCH", "DELETE"]
+    cached_methods         = ["GET", "HEAD", "OPTIONS"]
+    target_origin_id       = "s3_one" #module.s3_bucket_cms.s3_bucket_bucket_regional_domain_name
```

Comment:
I think this is a typo

Reply:
no, "s3_one" i correct....is a orign defined previously. with last commit removed comment "#module.s3_bucket_cms.s3_bucket_bucket_regional_domain_name"